tonic/models/vic/vic.py

```diff
@@ -95,11 +101,28 @@ def _call_vic(self, *args, **kwargs):
 
         vic_args = []
 
-        if "mpi_proc" in kwargs.keys():
-            if not isinstance(kwargs["mpi_proc"], int):
-                raise TypeError("number of processors must be specified as an integer")   
- 
-            vic_args.extend(['mpirun', '-np', '%.0d' % kwargs["mpi_proc"]])  
+        # Get mpi info
+        mpi_proc = kwargs.pop('mpi_proc', None)
+        if mpi_proc is not None:
+            if not isinstance(mpi_proc, int):
+                raise TypeError("number of processors must be specified as an"
+                                "integer")
+            vic_args.extend(['mpirun', '-np', '%.0d' % kwargs["mpi_proc"]])
+
+        # Get valgrind info
+        valgrind = kwargs.pop('valgrind', None)
+        if valgrind is not None:
+            if valgrind is True:
```

Comment:
good catch.  I'll update this slightly to protect against the `valgrind=False` case.  `valgrind` can either be a `bool` or a `str`, both of which can evaluate to `True` so that's why we have two checks here.